packages/oui-datagrid/src/datagrid.directive.js

```diff
@@ -19,7 +19,9 @@ export default () => ({
         onRowSelect: "&",
         onPageChange: "&",
         onCriteriaChanged: "&onCriteriaChange",
-        onSortChange: "&"
+        onSortChange: "&",
+        refreshable: "@?",
```

Comment:
Since it's a boolean, this should be `refreshable: "<?"`.
See for example the `customizable` attribute.

In the same way, we should support `<oui-datagrid refreshable></oui-datagrid>` as a `true` value. We have a helper addBooleanParameter for that: https://github.com/ovh-ux/ovh-ui-angular/blob/master/packages/oui-datagrid/src/datagrid.controller.js#L80